lua/mason-null-ls/mappings/source.lua

```diff
@@ -13,6 +13,8 @@ local null_ls_to_package = {
 	['lua_format'] = 'luaformatter',
 	['ansiblelint'] = 'ansible-lint',
 	['deno_fmt'] = 'deno',
+	['kotlin_debug_adapter'] = 'kotlin-debug-adapter',
```

Comment:
This shouldn't be needed. It follows the automatic look up logic

Reply:
https://github.com/jay-babu/mason-null-ls.nvim#auto-resolve-logic

Reply:
Ah, i thought I was being clever and getting ahead of the game!